.circleci/main.yml

```diff
@@ -138,7 +138,7 @@ jobs:
           path: /tmp/test-results
   helm-tests:
     docker:
-      - image: cimg/go:1.18.3
+      - image: cimg/go:1.18.4
```

Comment:
I noticed some things in this file are still go 1.17, and I'm not sure if that's for a good reason.  Would be good to try upgrading those at some point, maybe in a follow-up PR.